File: 3379-ScoreOfAString/3379-ScoreOfAString.js
// Last updated: 10/03/2026, 10:02:33
/**
 * @param {string} s
 * @return {number}
 */
var scoreOfString = function (str) {
    let score = 0;

    for (let i = 1; i < str.length; i++) {
        score += Math.abs(
            str.charCodeAt(i) - str.charCodeAt(i - 1)
        );
    }

    return score;
};

File: 3379-ScoreOfAString/3379-ScoreOfAString.cpp
// Last updated: 03/08/2026, 10:06:09
class Solution {
public:
    int scoreOfString(string s) {
        int score=0;
        for(int i=1;i<s.size();i++){
            score += abs(s[i]- s[i-1]);
        }
        return score;
    }
};